lib/v2/nrta/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
```

Comment:
This route should be added into `lib/v2/gov` as `.gov`-related website are added there.